src/sass/components/_anchor-preview.scss

```diff
@@ -0,0 +1,24 @@
+.medium-editor-anchor-preview {
+    font-family: $font-sans-serif;
+    font-size: 16px;
```

Comment:
Same as toolbar, the main container. I think we should leave it for now, but font-sizes and margin/paddings is something I would like to improve.